### PR TITLE
Glenn updates

### PR DESCRIPTION
**Pull request by ggailey777**
@BigMorty and @paulyuk

### CODE REVIEW

File: test.http [hunk 1]
```diff
@@ -0,0 +1,11 @@
+GET http://localhost:7071/api/httpget?name=Awesome%20Developer HTTP/1.1
```
Comment (paulyuk) [inline test.http:1]:
```suggestion
GET http://localhost:7071/api/httpget?name=World HTTP/1.1
```

**Review by paulyuk — CHANGES_REQUESTED** [review]
LGTM, tiny suggestion

File: README.md [hunk 2]
```diff
@@ -13,97 +13,129 @@ urlFragment: functions-quickstart-python-azd
...
+  + [Azure Functions extension](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azurefunctions)
 
 ### Get repo on your local machine
-Run the following GIT command to clone this repository to your local machine.
-```bash
-git clone https://github.com/Azure-Samples/functions-quickstart-python-azd.git
-```
 
-## Run on your local environment
+You can download this project template in one of these ways:
```
Comment (paulyuk) [inline README.md:29]: I like this but I think it's making it a little too long.  I would show only 1 or possibly 2 methods.  To me either azd init (no git) or fork and clone.

File: README.md [hunk 3]
```diff
@@ -13,97 +13,129 @@ urlFragment: functions-quickstart-python-azd
...
+## Run your app from the terminal
 
-3) Test the HTTP POST trigger in a new terminal window:
-```bash
-curl -i -X POST http://localhost:7071/api/http_post -H "Content-Type: text/json" -d '{"name":"yourname"}'
-```
+1. Run these commands in the virtual environment:
+
+    ```bash
+    pip install -r requirements.txt
```
Comment (paulyuk) [inline README.md:89]:
```suggestion
    pip3 install -r requirements.txt
```

pip3 and python3 work better multi plat.  pip and python only aliases work best on only Windows.

**Review by paulyuk — CHANGES_REQUESTED** [review]
Just a few comments

File: README.md [hunk 4]
```diff
@@ -13,97 +13,129 @@ urlFragment: functions-quickstart-python-azd
...
+    azd init --template functions-quickstart-python-http-azd
+    ```
+
++ Clone this repository locally using the `git clone` command:
+
+    ```bash
+    git clone https://github.com/Azure-Samples/functions-quickstart-python-azd.git
+    cd functions-quickstart-python-azd
+    ```
+
```
Comment (ggailey777) [inline README.md:43]:
```suggestion
```

**Review by paulyuk — APPROVED** [review]
LGTM.  thank you @ggailey777

**Pull request merged**